app/views/admin/_history.html.arb

```diff
@@ -62,7 +62,12 @@ div class: "header", id: "header" do
   end
 end
 
-panel "&nbsp;&nbsp;#{assigns[:versioned_object].class.name.humanize} History: #{assigns[:versioned_object].title}".html_safe do
+verion_object_name = if "#{assigns[:versioned_object].class.name}" == 'School'
```

Comment:
not a problem, just a small typo -- "s" missing from "version"

Reply:
@daryanypl updated typo. If you are good with PR. Can you approve.